Makefile

```diff
@@ -1,27 +1,28 @@
-egg-info:
-	pip install -q -e .
+app:
+	gunicorn -w 5 -b 0.0.0.0:8000 --log-level info --log-file /var/log/gunicorn.log aleph.manage:app
```

Comment:
Yup I think the Makefile should be a collection of all our commands (sort of how npm scripts are for JS projects). There will be an equivalent for running Flask in debug mode.